test/plugin/test_out_redis.rb

```diff
@@ -120,6 +121,22 @@ def test_write_with_insert_key_prefix
       assert_equal "3", d.instance.redis.hget("test.#{time}.1", "a")
     end
 
+    def test_write_with_ttl
+      d = create_driver CONFIG + %[
+        ttl 60
+      ]
+      time = event_time("2011-01-02 13:14:00 UTC")
```

Comment:
This is because default key creating step uses integer time value: https://github.com/fluent-plugins-nursery/fluent-plugin-redis/pull/19/files#diff-8ccea6604fe322ac050ee836dc41f81dR79
And using time in key prevents wrong assertion when we running multiple times of test cases